### PR TITLE
Helmfile.d - Add stable/grafana

### PR DESCRIPTION
**Pull request by alebabai**
## what
* add helmfile with stable/grafana chart
* add values with examples how to add custom dashboards (for stable/grafana and kube-prometheus/grafana)
 
## why
* Dashboards import is unclear with default kube-prometheus grafana deployment

### CODE REVIEW

File: helmfile.d/0420.grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+helmDefaults:
+  args:
+    - "--wait"
+    - "--recreate-pods"
```
Comment (osterman) [inline helmfile.d/0420.grafana.yaml]: Drop this as it causes outages.

File: helmfile.d/0410.kube-prometheus.yaml [hunk 2]
```diff
@@ -40,6 +40,7 @@ releases:
   chart: "coreos-stable/kube-prometheus"
   version: "0.0.66"
   values:
+    - "values/kube-prometheus.grafana.dashboards.yaml"
     - global:
         rbacEnable: false
```
Comment (osterman) [inline helmfile.d/0410.kube-prometheus.yaml]: Make this an env.

**Review by osterman — CHANGES_REQUESTED** [review]
See comments.

File: helmfile.d/0420.grafana.yaml (hunk 1 above)
Comment (osterman) [inline helmfile.d/0420.grafana.yaml]: Drop this as it causes outages.

File: helmfile.d/0420.grafana.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+#######################################################################################
+
+- name: "grafana"
+  namespace: "monitoring"
+  labels:
+    chart: "grafana"
+    component: "monitoring"
+    namespace: "monitoring"
+    vendor: "kubernetes"
+    default: "true"
```
Comment (aknysh) [inline helmfile.d/0420.grafana.yaml]: default: "false"

File: helmfile.d/0420.grafana.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        size: 10Gi
+        accessModes:
+          - ReadWriteOnce
+      datasources:
+        datasources.yaml:
+          apiVersion: 1
+          datasources:
+            - name: Prometheus
+              type: prometheus
+              url: 'http://kube-prometheus-prometheus:9090'
```
Comment (aknysh) [inline helmfile.d/0420.grafana.yaml]: make it a variable:
url:  {{ env "GRAFANA_PROMETHEUS_ENDPOINT" | default "http://kube-prometheus-prometheus:9090" }}'

**Comment by aknysh** [comment]
add this section:

```

serviceAccount:
  create: true
  name:

```

Make `create` and `name` variables
https://github.com/helm/charts/blob/master/stable/grafana/values.yaml#L4

**Comment by aknysh** [comment]
add `ingress` section similar to this:

```

ingress:
      enabled: {{ env "GRAFANA_INGRESS_ENABLED" | default "false" }}
      annotations:
        kubernetes.io/ingress.class: "nginx"
        kubernetes.io/tls-acme: "true"
        external-dns.alpha.kubernetes.io/ttl: "60"
        external-dns.alpha.kubernetes.io/target: {{ env "GRAFANA_INGRESS" }}
        external-dns.alpha.kubernetes.io/hostname: {{ env "GRAFANA_HOSTNAME" }}
      hosts:
      - {{ env "GRAFANA_HOSTNAME" }}
      tls:
      - secretName: {{ env "GRAFANA_SECRET_NAME" | default "grafana-server-tls" }}
        hosts:
        - {{ env "GRAFANA_HOSTNAME" }}

```

File: helmfile.d/0420.grafana.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    chart: "grafana"
+    component: "monitoring"
+    namespace: "monitoring"
+    vendor: "kubernetes"
+    default: "true"
+  chart: "stable/grafana"
+  version: "1.12.0"
+  values:
+    - "values/grafana.dashboards.yaml"
+    - persistence:
```
Comment (aknysh) [inline helmfile.d/0420.grafana.yaml]: make it a variable

File: helmfile.d/0420.grafana.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        enabled: true
+        size: 10Gi
```
Comment (aknysh) [inline helmfile.d/0420.grafana.yaml]: make `size` a variable

**Review by aknysh — DISMISSED** [review]
inline comments

**Comment by osterman** [comment]
This will fix CI: https://github.com/cloudposse/helmfiles/pull/7

File: helmfile.d/0420.grafana.yaml (hunk 1 above)
Comment (alebabai) [reply helmfile.d/0420.grafana.yaml]: did you mean only "--recreate-pods" or both?

**Review by osterman — APPROVED** [review]

**Pull request merged**